### ISSUE TITLE
Eve Data

### ISSUE THREAD

**Issue by timthedevguy** [binarygod/AmSYS#13]
Need to merge EveBundle Entities to AppBundle so that Doctrine can provide relationship mapping.

**Comment by Aarondorn2** [comment]
This is an interesting issue.  I was going to put the 'ContactEntity' in the EveBundle as it makes more sense there, but couldn't due to the need to relationally map it to AuthorizationEntity.  I did put a 'ContactModel' class in EveBundle, along with the Eve API classes.  

Are their any other solutions for cross-bundle relational mapping?

**Comment by binarygod** [comment]
I researched for some time on this and couldn't find a way.  I tried moving everything over awhile ago and just changed references and it failed horribly!

**Comment by Aarondorn2** [comment]
lol. what if we mov eeverything into one bundle and then just organize into logical folders like discussed before?

entities
-amsys
-eve
controllers
-amsys
model
-amsys
-eve


etc.

**Comment by binarygod** [comment]
Yeah thats what I had tried doing, I probably just missed references or something.  But yes, this is a good idea!
